airbyte-api-server/src/main/java/io/airbyte/api/server/Application.java

```diff
@@ -4,11 +4,22 @@
 
 package io.airbyte.api.server;
 
+import dev.speakeasyapi.micronaut.EnableSpeakeasyInterceptor;
+import dev.speakeasyapi.micronaut.SpeakeasyFilter;
+import io.micronaut.context.annotation.Import;
 import io.micronaut.runtime.Micronaut;
 
+@Import(classes = { SpeakeasyFilter.class, EnableSpeakeasyInterceptor.class }, annotated = "*")
 public class Application {
 
   public static void main(final String[] args) {
+    if (args.length > 1) {
```

Comment:
can we add a comment here to explain what the expected args are to setup speakeasy?